scripts/release.sh

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
+#
+# Name::        release.sh
+# Description:: Use this script to prepare a new release on Github,
+#               the automation will create a GH tag like 'v0.1.0'
+#               (using the VERSION file)
+# Author::      Salim Afiune Maya (<afiune@lacework.net>)
+#
+set -eou pipefail
+
+readonly org_name=lacework
+readonly project_name=terraform-kubernetes-agent
+readonly git_user="Alan Nix"
+readonly git_email="alan.nix@lacework.net"
```

Comment:
Let us start using our Lacework github account for these commits:
```suggestion
readonly git_user="Lacework Inc."
readonly git_email="ops+releng@lacework.net"
```